src/pages/index.md

```diff
@@ -2,10 +2,10 @@
 template: home-page
 path: /
 hero:
-  title: Stake on Ethereum’s first private computer and earn up to
+  title: tBTC unlocks Bitcoin on DeFi.
+  subtitle: Put your Bitcoin to work, without CeFi.
```

Comment:
Non-blocking:

We don't use `tickers` anymore (see line 16 on this file). Maybe we could remove it?

If you agre then also remove the
```
tickers {
            label
          }
```
from the home-page graphql query.